README.md

```diff
@@ -435,3 +435,16 @@ unsafe { println!("{}", myUnion.test };
 ```
 
 This is required as the rust compiler can not guarantee the safety of the union, or that the member being addressed exists. The union RFC is [here](https://github.com/tbu-/rust-rfcs/blob/master/text/1444-union.md) and the tracking issue is [here](https://github.com/rust-lang/rust/issues/32836).
+
+### Crate name overrides
+
+`gir` uses simple rule to convert namespace to crate name and it sometimes go wrong, Ex. "WebKit2WebExtension" namespace converted to "web_kit2_web_extension" crate name that looks bad.
+
+To fix it table `crate_name_overrides` can be used.
+
+This also replaces name FFI crates.
```

Comment:
```
This also replace FFI crates' name.
```

Reply:
"'" really needed?

Reply:
Yes, it's to mark "possession". But if someone else could confirm it maybe?

Reply:
Thanks, I didn't know that this case is possible ["We use ’s after a singular noun and ’ after a plural noun."](https://dictionary.cambridge.org/grammar/british-grammar/determiners/possession-john-s-car-a-friend-of-mine)